### PR TITLE
11.0 42288143e4765d33a21e8e4c12fe55c3bdf41d78

### PR DESCRIPTION
**Pull request by Rusllan**
Merging 10.0 in 11.0

### CODE REVIEW

File: pos_product_available_negative/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    'data': [
+        'data.xml',
+        'views.xml',
+    ],
+
+    "price": 50.00,
+    "currency": "EUR",
+
+    'installable': True,
+}
```
Comment (yelizariev) [inline pos_product_available_negative/__openerp__.py]: Wrong

File: pos_multi_session/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,8 @@
+`4.2.3`
+-------
+
+- **FIX:** Don't make automatic scrolling of the current order on changing other orders
```
Comment (yelizariev) [inline pos_multi_session/doc/changelog.rst]: Someone has to check that it works in odoo 11

File: pos_mobile_restaurant/doc/changelog.rst [hunk 3]
```diff
@@ -1,3 +1,8 @@
+`1.3.7`
+-------
+
+- **FIX:** The menu is not opened after changing a floor
```
Comment (yelizariev) [inline pos_mobile_restaurant/doc/changelog.rst:9]: Someone has to check that it works in odoo 11

File: pos_mobile/doc/changelog.rst [hunk 4]
```diff
@@ -1,3 +1,10 @@
+`1.1.8`
+-------
+
+- **FIX:** Font size of buttons
+- **FIX:** Order list scrolling
+- **FIX:** Scrollbar of Products after opening bottom menu
```
Comment (yelizariev) [inline pos_mobile/doc/changelog.rst:17]: Someone has to check that it works in odoo 11

File: pos_debt_notebook_sync/doc/changelog.rst [hunk 5]
```diff
@@ -1,3 +1,8 @@
+`1.1.1`
+-------
+
+- **FIX:** Endless update requests on the customer screen after renamings in pos_longpolling
```
Comment (yelizariev) [inline pos_debt_notebook_sync/doc/changelog.rst:4]: Someone has to check that it works in odoo 11

File: pos_debt_notebook/doc/changelog.rst [hunk 6]
```diff
@@ -1,3 +1,15 @@
+`5.1.0`
+-------
+
+- **NEW:** Wizard allow to create invoices for sets of partners: `Balance update`, `New Balance`
+- **FIX:** Error related to paying with not discount credit journals in POS
+- **FIX:** Invoices paid with credit journals now counts in partner debts
+
+`5.0.3`
+-------
+
```
Comment (yelizariev) [inline pos_debt_notebook/doc/changelog.rst:9]: Someone has to check that it works in odoo 11

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
:+1:  **FIX:** Don't make automatic scrolling of the current order on changing other orders

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
:+1: **FIX:** The menu is not opened after changing a floor

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
pos_mobile
:+1:  **FIX:** Order list scrolling
:+1:  **FIX:** Scrollbar of Products after opening bottom menu

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
pos_mobile
:-1:  **FIX:** Font size of buttons

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
:+1:  **FIX:** Endless update requests on the customer screen after renamings in pos_longpolling

**Review by ilmir-k — CHANGES_REQUESTED** [review]
Validation error when trying to add a partner to _Write-off Credits_ wizard

File: hw_twitter_printing/__manifest__.py [hunk 7]
```diff
@@ -4,11 +4,11 @@
     "category": "Point of Sale",
     # "live_test_URL": "",
     "images": [],
-    "version": "1.0.0",
+    "version": "10.0.1.0.0",
```
Comment (Ramil-Mukhametzyanov) [inline hw_twitter_printing/__manifest__.py]: should be 11.0.1.0.0

File: pos_multi_session/doc/changelog.rst (hunk 2 above)
Comment (Ramil-Mukhametzyanov) [reply pos_multi_session/doc/changelog.rst]: > Someone has to check that it works in odoo 11

:+1:

File: pos_mobile/doc/changelog.rst (hunk 4 above)
Comment (Ramil-Mukhametzyanov) [reply pos_mobile/doc/changelog.rst:17]: :+1:

File: pos_debt_notebook_sync/doc/changelog.rst (hunk 5 above)
Comment (Ramil-Mukhametzyanov) [reply pos_debt_notebook_sync/doc/changelog.rst:4]: :+1:

File: pos_debt_notebook/doc/changelog.rst (hunk 6 above)
Comment (Ramil-Mukhametzyanov) [reply pos_debt_notebook/doc/changelog.rst:9]: :+1:  **NEW:** Wizard allow to create invoices for sets of partners: `Balance update`, `New Balance`
:+1:  **FIX:** Error related to paying with not discount credit journals in POS
:+1:  **FIX:** Invoices paid with credit journals now counts in partner debts

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Review by GabbasovDinar — APPROVED** [review]

**Pull request merged**